packages/account/src/Components/poi-manual-upload-failed/poi-manual-upload-failed.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { ReactNode } from 'react';
+import { Icon } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import IconMessageContent from '../icon-message-content';
+
+type TPOIManualUploadFailed = {
+    error: string;
+    message?: ReactNode;
+};
+const POIManualUploadFailed = ({ children, message, error }: React.PropsWithChildren<TPOIManualUploadFailed>) => (
+    <IconMessageContent
+        message={message ?? <Localize i18n_default_text={'Proof of identity documents upload failed'} />}
+        text={error}
+        icon={<Icon icon='IcPoiFailed' size={128} />}
+        className='account-management-dashboard'
+    >
+        {children && <>{children}</>}
```

Comment:
```suggestion
        {children}
```
Why conditionally check to add empty fragment?